packages/server/api/controllers/uidreview.controller.js

```diff
@@ -0,0 +1,44 @@
+const knex = require('../../config/db');
+const HttpError = require('../lib/utils/http-error');
+
+const getReviewByUid = async (id, uid) => {
+  if (!id) {
```

Comment:
You could also do a check if id is a number and not string. Like isNan

Reply:
OK! I change it to isNaN()